shared/src/ledger/protocol/transactions/ethereum_events/mod.rs

```diff
@@ -442,4 +418,61 @@ mod tests {
              voting power so far"
         );
     }
+
+    #[test]
+    /// Assert we don't return anything if we try to get the votes for an empty
+    /// vec of events
+    pub fn test_get_votes_for_events_empty() {
```

Comment:
We should have a test where a message has multiple copies of a sig from the same validator.

Reply:
that should be written for #760